Formula/helmfile.rb

```diff
@@ -0,0 +1,30 @@
+class Helmfile < Formula
+  desc "Deploy Kubernetes Helm Charts"
+  homepage "https://github.com/roboll/helmfile"
+  url "https://github.com/roboll/helmfile/archive/v0.19.0.tar.gz"
+  sha256 "27793534046d32228391ce80bc11396ac0587607b79f168eaed0d71cf6472955"
+
+  depends_on "go" => :build
+  depends_on "kubernetes-helm"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/roboll/helmfile").install buildpath.children
+    cd "src/github.com/roboll/helmfile" do
+      system "go", "build", "-ldflags", "-X main.Version=v#{version}",
+             "-o", bin/"helmfile", "-v", "github.com/roboll/helmfile"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    (testpath/"helmfile.yaml").write <<-EOS
+      repositories:
+      - name: stable
+        url: https://kubernetes-charts.storage.googleapis.com/
+    EOS
+    system "helm", "init", "--client-only"
```

Comment:
`system bin/"helm", "init", "--client-only"`

Reply:
Oh, unless `helm` is part of `kubernetes-helm`?

Reply:
Correct, `helm` is the main binary `kubernetes-helm` installs.

Reply:
Eh. Gonna throw this one to @ilovezfs for a ruling then. Personally I prefer when we reference other formulae we do things like this:

```ruby
system Formula["kubernetes-helm"].opt_bin/"helm", "init", "--client-only"
```
but since Joe runs this place these days I'll let him decide on how fussy he wants to be.

Reply:
I certainly don't mind being fussy. Just pushed the change.

Reply:
I don't think it's an absolutely mandatory thing in general; I just prefer the longhand because it's clearer & slightly more resilient. Thanks for the change ❤️